internal/runner/rmq.go

```diff
@@ -343,38 +369,62 @@ var (
 	qCheck   sync.Once
 )
 
+func generateMgtRMQ(amqpURL string, amqpMgtURL string) (mgtURL amqp.URI, err kv.Error) {
+	if len(amqpURL) == 0 && len(amqpMgtURL) == 0 {
+		return mgtURL, kv.NewError("amqp-url, or amqp-mgt-url was not specified on the command line, or as an env var, cannot start rabbitMQ").With("stack", stack.Trace().TrimRuntime())
+	}
```

Comment:
That's my lack of Golang knowledge, but what is mgtURL here? If it's named return value, is it initialized at this code point?

Reply:
Yes it is a named return value. Returned values are initialized to their 'nil' value when the function is entered.